client/.eslintrc.json

```diff
@@ -27,7 +27,7 @@
         ],
         "linebreak-style": [
             "error",
-            "unix"
+            "windows"
```

Comment:
[This](https://stackoverflow.com/a/53110172) might be useful. But that solution assumes a .js file......

Reply:
This change was to fix an error I got when I updated the repository.

error  Expected linebreaks to be 'LF' but found 'CRLF'"  linebreak-style

Reply:
Okay I edited the editorconfig file to auto make vscode use lf as the line ending